python/tests/test_highlevel.py

```diff
@@ -851,6 +851,52 @@ def test_minlex_postorder_multiple_roots(self):
         assert tree_orders == expected_result
 
 
+class TestMRCA:
+    t1 = tskit.Tree.generate_balanced(3)
+    #  4
+    # ┏━┻┓
+    # ┃  3
+    # ┃ ┏┻┓
+    # 0 1 2
+
```

Comment:
When I'm reading the tests I need to keep in mind what t1 and t2 are. Unless there's a particular property of the 2 sample tree we're using, I think it'd be easier to just have a single tree that we're using as an example.